modules/apps/commerce/commerce-product-service/src/main/java/com/liferay/commerce/product/internal/upgrade/registry/CommerceProductServiceUpgradeStepRegistrator.java

```diff
@@ -406,6 +406,10 @@ protected String[][] getTableAndPrimaryKeyColumnNames() {
 			UpgradeProcessFactory.alterColumnName(
 				"CPOption", "DDMFormFieldTypeName",
 				"commerceOptionTypeKey VARCHAR(75) null"));
+		registry.register(
+				"5.10.0", "5.11.0",
+				new com.liferay.commerce.product.internal.upgrade.v5_7_0.
+						CPDefinitionLinkUpgradeProcess(_assetEntryLocalService));
```

Comment:
This looks wrong, I think you should use the newly created upgrade process instead